doc/source/ray-core/examples/batch_training.ipynb

```diff
@@ -557,13 +591,30 @@
     "    )\n",
     "    load_tasks = [read_into_object_store_pg.remote(file) for file in files]\n",
     "    group_refs = {}\n",
-    "    for i, refs in enumerate(ray.get(load_tasks)):\n",
-    "        group_refs[files[i]] = refs\n",
+    "    for location_id, refs in enumerate(ray.get(load_tasks)):\n",
```

Comment:
This should be `file_id` instead of `location_id`. Though in general it'd be helpful to give all the `refs` more descriptive names so it's clear what is the object(s) being referenced.